### ISSUE TITLE
When closing the app it will send a contact request notification

### ISSUE THREAD

**Issue by LFGaming** [berty/berty#4643]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Berty product

Mobile app

### Berty product version

testflight version 2.460.0 (4621)

### OS

iOS

### OS version

iOS 15.6.1

### Device

iPhone 12 mini

### Steps to reproduce

1. Open the app.
2. Close the app.
3. You will get an notification.
4. See video

(maybe the reason)
I send a contact request to an android phone, that never got recieved and it looks like i get this notification since then.

### Current behavior

https://user-images.githubusercontent.com/38732192/227365447-a8c0b4f9-d6e8-46d5-8d1b-713e351e5ea0.MP4

**Comment by LFGaming** [comment]
> @LFGaming . As you know, the latest TestFlight has new notification changes. Is this still a bug?

Yes sort off. When I make a new contact request it will give me the notification a few more times even when accepted. After that (no idea how many times but I can test that at some point) it's gone.

**Comment by LFGaming** [comment]
Hi @jefft0,
After a bit more playing around I got this:
When closing the app while in the settings (or any other screen exept the home page so for example a chat too) I recieve a "new" contact request message, this then sends me the notification in the app. It seems to me that the problem lays somewhere in that problem.
I have added a video here to hopefully make it more clear:


https://github.com/berty/berty/assets/38732192/0ef3de90-f80f-45fc-b808-03dd86549966

**Comment by jefft0** [comment]
Would need to update the messenger database to mark a message as already notified. Maybe not simple to implement. Keep this open, but put in backlog for when we can investigate.

**Comment by SohamAnd** [comment]
i would like to contribute in this project and to solve this issue.
